components/subViews/company/partnerContent.vue

```diff
@@ -3,7 +3,7 @@
 
 
     <!-- YOUTUBE VIDEO -->
-    <div v-if="post.sponsor_youtube_video" class="video">
+    <div v-if="post.sponsor && post.sponsor_youtube_video" class="video">
```

Comment:
I would prefer to have the v-if on a higher level. Either on div.partnercontent or in the _companies.vue when the component is called.